js/data/stations.ts

```diff
@@ -0,0 +1,39 @@
+import { Station } from "../models/station";
+
+const buildStation = (id: string, name: string, spacing: number): Station => ({
+  id,
+  name,
+  spacing,
+});
+
+export const alewifeAndrewStationsArray: Station[] = [
```

Comment:
suggestion: `alewifeAndrewStationsArray` is kind of weird name, could this instead be something more akin to this?
```
export const StationSets = {
  AlewifeAndrew: [
     buildStation(...),
     buildStation(...),
  ],
  JFKAshmont: [
     ...
  ],
  JFKBraintree: [
     ...
    ],
};
```

And then referred to as `StationSets.AlewifeAndrew`?